src/components/alert/alert.stories.js

```diff
@@ -28,7 +28,7 @@ const handleOpen = () => {
   action('open')();
 };
 
-function makeStory(name, themeSelector, disableChromatic = false) {
+function makeStory(name, themeSelector, disableChromatic = true) {
```

Comment:
I think in this case the `disableChromatic` arg along with
`chromatic: {
      disable: disableChromatic
    }`
can be removed.

Reply:
Or am I wrong here @DlgSHi ?

Reply:
The code change is ok, we don't need `chromatic` check for this story.